src/core/column/column.view.js

```diff
@@ -46,6 +46,13 @@ export class ColumnView extends View {
 				service.invalidate('column.view', e.changes, PipeUnit.column);
 			}
 		});
+
+		model.filterChanged.watch(e => {
```

Comment:
i think this should be in headview itself without calling pipe in other case u use two different approaches: pipe and headview.invalidate